core/descriptors/src/org/jetbrains/kotlin/resolve/scopes/MemberScope.kt

```diff
@@ -46,6 +46,8 @@ interface MemberScope : ResolutionScope {
             p.println("Empty member scope")
         }
 
+        override fun definitelyDoesNotContainName(name: Name): Boolean = true
```

Comment:
What is this change for?

Reply:
There are empty member scopes created in REPL scripts, so it helps us to filter them out